docs: Add manual metric load test (#757)
kyma-project/telemetry-manager: docs/contributor/telemetry-load-test/README.md

- curl 8.4.x
- jq 1.6

## Test Script

> All test scenarios use a single test script [run-load-test.sh](assets/run-load-test.sh), which provides four parameters: '`-t` for test type supported values are `traces, metrics, metricagent`, `-n` for test name e.g. `0.92 Traces`,`-m` for multi pipeline scenarios, and `-b` for backpressure scenarios

Reviewer: ```suggestion
All test scenarios use a single test script [run-load-test.sh](assets/run-load-test.sh), which provides the following parameters:
- '`-t` for test type, supported values are `traces, metrics, metricagent`
- `-n` for test name, such as `0.92 Traces`
- `-m` for multi pipeline scenarios
- `-b` for backpressure scenarios
```

Author: This section updated, please review again

## Metrics Test

> Metric test consists of two main test scenarios. The first scenario tests the Metric Gateway KPIs, and the second one tests Metric Agent KPIs.

Reviewer: ```suggestion
The metrics test consists of two main test scenarios. The first scenario tests the Metric Gateway KPIs, and the second one tests Metric Agent KPIs.
```

> The tests are executed for 20 minutes for each test case to have a stabilized output and reliable KPIs. 

Reviewer: ```suggestion
The tests are executed for 20 minutes, so that each test case has a stabilized output and reliable KPIs. 
```

> Contrast to Metric Gateway test, Metric Agent test deploy passive metric producer ([Avalanche Prometheus metric load generator](https://blog.freshtracks.io/load-testing-prometheus-metric-ingestion-5b878711711c)) and metrics will be scraped by Metric Agent from the producer.

Reviewer: ```suggestion
In contrast to the Metric Gateway test, the Metric Agent test deploys a passive metric producer ([Avalanche Prometheus metric load generator](https://blog.freshtracks.io/load-testing-prometheus-metric-ingestion-5b878711711c)) and the metrics are scraped by Metric Agent from the producer.
```

> Test setup deploy 20 individual metric producer pods each of this producer produces 1000 Metrics with 10 metric series, Metric Agent collect metrics via Pod scraping as well as Service scraping to test both Metric Agent receivers configuration.

Reviewer: ```suggestion
The test setup deploys 20 individual metric producer Pods; each which produces 1000 metrics with 10 metric series. To test both Metric Agent receiver configurations, Metric Agent collects metrics with Pod scraping as well as Service scraping.
```
